src/Utils.ts

```diff
@@ -986,12 +986,129 @@ function setDummyInputDebugValue(
     );
 }
 
+export class DummyInputObserver implements Types.DummyInputObserver {
+    private _win?: GetWindow;
+    private _offsetsQueue: (() => void)[] = [];
+    private _offsetsTimer?: number;
```

Comment:
I would prefer `resetDummyInputs` rather than `offset` since the current name really has no relevance to the operation that is happening